src/components/ui/AspectRatio/AspectRatio.tsx

```
@@ -0,0 +1,25 @@
+
+export type AspectRatioProps = {
+    children: React.ReactNode;
+    customRootClass?: string;
+    className?: string;
+    ratio?: string;
+    props: Record<string, any>[];
+}
+
+const AspectRatio = ({ children, customRootClass, className, ratio="1", ...props }: AspectRatioProps) => {
+
+    if (isNaN(Number(ratio)) && !ratio.match(/^(\d+)\/(\d+)$/)) ratio = "1"
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)